### DISCUSSION TITLE
Add a space between "phone" and "number"

### DISCUSSION THREAD

**Discussion by bickelj** [dasniko/keycloak-extensions-demo#12] [Ideas]
Thank you for all of your great examples. They have been very helpful. In the required action example (update mobile number), it prints the word "phonenumber" without a space between "phone" and "number." I think it would help to add the space. Cheers!

More specifically, lines [1 and 3 here](https://github.com/dasniko/keycloak-extensions-demo/blob/bd79483548b0eebb935505b917c8dcf265160d4b/requiredaction/src/main/resources/theme-resources/messages/messages_en.properties#L1).

**Comment by kfogel** [comment]
This looks like a straightforward typo fix to me, FWIW -- IMHO just make it a PR?  (I'm happy to do so if you want, but I figured it's your fix so better if the PR comes from you, @bickelj).

**Comment by dasniko** [comment]
Guys, thanks for that hint. But be aware that this is an example and demo repository for my Keycloak extensions and use cases. It claims not to be perfectly fine for production usage. If you want to use it, fine, clone or copy it, it's free under the Apache 2 license. You might want to fix some typos in your distribution.
Perhaps I will fix these typos some time in the future, when I'll rework the respective extension, but please understand that I won't merge any PRs here. Thanks.

**Reply by kfogel** [reply]
Ah, ok.  Thanks for the reply, @dasniko.
